### ISSUE TITLE
Bug report for trillium-static-compiled: Missing charset in response headers

### ISSUE THREAD

**Issue by mathiversen** [trillium-rs/trillium#105]
When i try to serve a `.js` file i get this error in firefox: `Uncaught SyntaxError: identifier starts immediately after numeric literal` and it looks like `charset=utf-8` is missing in the content-type.

Not sure if you can just add the charset here: [static-compiled](https://github.com/trillium-rs/trillium/blob/333ffb22bf6199f1fd649dd04901981d657f0fd5/static-compiled/src/lib.rs#L115) or if you want to do something else.

In short i think it should be: `Content-Type: "application/javascript; charset=utf-8"`

**Comment by jbr** [comment]
Thanks for filing this! Will get a patch release out today

**Comment by jbr** [comment]
Closed by #106 and released as trillium-static-compiled v0.3.1
